config/application.yml.default

```diff
@@ -199,6 +199,7 @@ min_password_score: 3
 minimum_wait_before_another_usps_letter_in_hours: 24
 mx_timeout: 3
 new_device_alert_delay_in_minutes: 5
+no_verify_by_mail_for_biometric_comparison_enabled: true
```

Comment:
I started to bikeshed the leading "no_" here but then read past comments and decided it's lovely. 😇